sdk/async_api/wallet_spot_trade_bust_update_payload.py

```
@@ -0,0 +1,10 @@
+from __future__ import annotations
+from typing import Any, List, Dict, Optional
```

Ruff (0.15.5)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)